concept/concept.md

```diff
@@ -0,0 +1,254 @@
+
+3. In a third step all the remaining Features will be implemented to build the full product. 
+
+### User Account
+
+- Sign-up
+- Agree to Data Privacy Statement
+- Agree to Terms of Service
+- Login
+- Logoff
+- Change User Name
+- Change Email Address
+- Change Password
+- Delete Account
+- Download User's Content
+- GDPR-Information about stored Content
+- Choosing Interface Language (e.g. German / English / French)
+
+### User  Profile
+
+- Upload and Change Avatar
+- Upload and Change Profile Picture
+- Edit Social Media Accounts
+- Edit Locale information
+- Show and delete Bookmarks (later)
+- Show Posts of a specific User
+- Show Comments of a specific User
+
+### Cockpit
+
+[Clickdummy](https://preview.uxpin.com/24a2ab8adcd84f9a763d87ed27251351225e0ecd#/pages/99768919/simulate/sitemap?mode=i)
+
+- Show Link to own Profile
+- Show Friends Widget
+- Show Favorites Widget
+- Show Get Friends Widget
+- Show popular Hashtags Widget
+- Show Mini-Statistics Widget (all time)
+- Show Chatrooms Widget
+- Show List of Let's Talk requests with online status of requesting people
+
+### Posts
+
+- Creating Posts
+- Upload Teaser Picture for Post
+- Upload additional Pictures
+- Editing Title and Content
+- Allow embedded Conten (Videos, Sound, ...)
+- Choosing a Category
+- Adding Tags
+- Choosing Language (e.g. German / English / French)
+- Choosing Visibility (Public / Friends / Private)
+- Shout Button for Posts
+- Bookmark Posts (later)
+- Optionally provide Let's Talk Feature
+- Optionally provide Commenting Feature
+
+### Comments
+
+- Creating Comments 
+- Deleting Comments
+- Editing Comments
+- Upvote comments of others
+
+### Article List
+
+- Show Posts by Tiles
+- Show Posts as List
+- Filter by Category (Health and Wellbeing, Global Peace & Non-Violance, ...) 
+- Filter by Mood (Funny, Happy, Surprised, Cry, Angry, ...)
+- Filter by Source (Connections, Following, Individuals, Non-Profits, ...)
+- Filter by Posts & Tools (Post, Events, CanDos, ...)
+- Filter by Format Type (Text, Pictures, Video, ...)
+- Extended Filter (Continent, Country, Language, ...) 
+- Sort Posts by Date
+- Sort Posts by Shouts
+- Sort Posts by most discussed
+- Sort Posts by most Emojis
+
+### Blacklist
+
+[Video](https://www.youtube.com/watch?v=-uDvvmN8hLQ)
+
+- Blacklist Users
+- Blacklist Subjects
+- Blacklist Tags
+- Switch on/off Adult Content
+
+### Search
+
+- Search for Categories
+- Search for Tags
+- Fulltext Search
+
+### CanDos
+
+- Creating CanDos
+- Editing Title and Content
+- Choosing a Category
+- Adding Tags
+- Choosing Language (e.g. German / English / French)
+- Choosing Visibility (Public / Friends / Private)
+- Choosing Difficulty
+- Editing Why - why should you do this
+- Editing Usefulness - what is it good for
+
+### Versus (interaction on existing Post)
+
+- Create / edit / delete Versus
+
+### Jobs
+
+- Create, edit and delete Jobs
```

Comment:
Jobs can be created by everyone?

Reply:
Yes. Clarified and changed in updated branch network-concept